src/components/getFromApi/services/AxiosConfig.jsx

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios'
+
+export default axios.create ({
+    headers: {
+        'x-rapidapi-key': '06680bfbb0msh40b9b35c3e0b9ebp1c368fjsn857cece02668',
+        'x-rapidapi-host': 'hotels4.p.rapidapi.com',
+        useQueryString: true
+    }
+})
```

Comment:
u can add this 
 axios.create({
  baseURL: 'https://hotels4.p.rapidapi.com',
});
 u don't need to repeat it every time

Reply:
Move all files to /src/services folder